libp2p/NodeTable.cpp

```diff
@@ -409,6 +434,7 @@ void NodeTable::onReceived(UDPSocketFace*, bi::udp::endpoint const& _from, bytes
             return;
         }
 
+        cnetdetails << "Received " << datagramNameFromType(packet->packetType()) << " from " << packet->sourceid << "@" << _from.address() << ":" << _from.port();
```

Comment:
I'd suggest outputting this and other your logs to `LOG(m_logger)` (it's lower verbosity (3 - Debug), but for me Debug verbosity is to log everything that happens, and 4 - Trace is for dumping into log actual binary contents of all data packets, and for tracing VM execution)

Also `m_logger` is more efficient than global `cnetdetails`, because it doesn't need to be thread-safe.